### DISCUSSION TITLE
E900v21E不能冷机启动

### DISCUSSION THREAD

**Discussion by A-nemon-e** [ophub/amlogic-s9xxx-armbian#1660] [Amlogic] [answered]
如题，我两个盒子一个刷了5.10一个5.15，使用同样的overload，5.15的那个没法冷启动，会卡在花屏处且每次卡的地方不一样（TTL焊盘被我搞坏了看不了卡哪了，跳线用的漆包线一时半会不在手上），试了20多次用风扇把cpu吹冷了以后第一次一定不能开机，趁热重新加电大概率能开，（已经在里面跑东西了懒得重刷别的版本了）有无遇到类似问题的让我走个捷径🆒🆒。。
E900v21E, S905L2, 5800-2ATA410, 1+8G

**Comment by ophub** [comment]
应该和温度无关。可能是因为你使用的5.10使用了主线u-boot，所以引导启动支持的好。

假设和温度低有关，那装风扇的人要多盖点被子了，太凉了不行✋

不用重装系统，更换内核并启用主线u-boot一条命令的事。

**Reply by A-nemon-e** [reply]
e900v21e是没有完整主线u-boot的，使用的都是您database里的overload。5.15的overload是我手动改了armbian-install以后强行装上的（因为他默认的need_overload是no，导致不识别网卡和内存识别很小）
那是不是两个盒子的bootloader不一样的原因呢？（原始安卓系统是不一样的）

**Comment by A-nemon-e** [comment]
邪门，已改成5.10，甚至尝试5.4，无果。又用u盘重装，无果。冷机（物理）第一次启动emmc必不能启动，u盘就可以。我还原成原版安卓再全部按照装那个正常盒子的来一遍。。。

**Reply by A-nemon-e** [reply]
无效。邪门了。这俩盒子只有emmc和内存芯片不一样。。。

**Comment by A-nemon-e** [comment, marked as answer]
已解决，刷了tools里MGV2000安卓包后，修改armbian-install 使他安装u-boot.emmc选择MGV2000对应的114号进行armbian-install（虽然我用的release的5.15包是s905l2_e900v21e的）。
推测就是原机安卓的bootloader问题。
我这样做只是因为没有找到e900v21e的刷机包，就尝试了下这样竟然可以。（这个包没wifi就没wifi吧反正mt7668主线内核也还没支持）
非常感谢O大和其他大佬的教程和文件。

**Reply by A-nemon-e** [reply]
新问题：刷入emmc后，不能从u盘启动u盘的armbian。。。

**Comment by ophub** [comment]
把emmc的/boot/u-boot.src改个名，emmc里面的系统就无法使用了，就从USB启动了

**Reply by A-nemon-e** [reply]
您说的是boot.scr嘛？我改这个没用，改了emmc_autoscript就不能启动了，但是插上u盘他又给我引导到emmc的系统。这是什么原理呢

**Comment by ophub** [comment]
这个魔百盒的u-boot应该是设置的emmc优先于u盘，所以emmc的系统正常时就不会从u盘启动，你可以破坏了emmc里的关键启动文件，就可以从u盘启动了。
